docs/trade_protocol/trade-protocol.md

```diff
@@ -1,39 +1,28 @@
-# Haveno Trade Protocol
+## Overview
 
-Described here is the desired protocol for Haveno. Note that the discussion is still ongoing, so this protocol could be changed/updated in the future.
+Haveno is a decentralized network where people meet to exchange XMR for fiat or other cryptocurrencies. There are no central entities involved and trades happen directly between traders, which are both required to deposit and lock some XMR until the trade is completed. In case of disagreement between traders, both of them can request the involvement of an arbitrator, which will resolve the dispute.
 
-## Roles
+This document provides a simplified overview of Haveno's trade protocol, but a PDF with the technical details is available: [trade-protocol.pdf](trade-protocol.pdf).
+
+## Protocol
+
+### Roles
 
-- **Buyer** - person buying XMR
-- **Seller** - person selling XMR
 - **Maker** - person making offer
 - **Taker** - person taking offer
 - **Arbitrator** - entity resolving possible disputes
 
-For each trade, a trader is a buyer or seller and a maker or taker.
+### Overview of a trade
 
-## Protocol
+We assume the maker is selling XMR and the taker is buying them in exchange of fiat:
+
+1. (optional) Maker **deposit funds** to Haveno wallet and wait for them to be unlocked (~20 minutes).
+2. Maker **creates an offer** and locks on Haveno the security deposit + the amount to be sent to the taker.
+3. Taker **accepts offer** and locks their security deposit to reserve it.
+4. A **2/3 multisignature wallet is created** between maker, taker and arbitrator. The arbitrator holds the third key, so that they can be summoned in case of disputes and can use their key to punish the misbehaving trader. If there is no dispute, the traders will complete the transaction without involving an arbitrator.
```

Comment:
"so that they can be summoned in case of disputes." the penalty applies if breaking protocol, otherwise it's just arbitration